internal/services/compute/orchestrated_virtual_machine_scale_set.go

```diff
@@ -1432,11 +1432,13 @@ func expandOrchestratedVirtualMachineScaleSetExtensions(input []interface{}) (ex
 		}
 		extensionType := extensionRaw["type"].(string)
 
+		autoUpgradeMinorVersion, _ := extensionRaw["auto_upgrade_minor_version_enabled"].(bool)
+
 		extensionProps := virtualmachinescalesets.VirtualMachineScaleSetExtensionProperties{
 			Publisher:                pointer.To(extensionRaw["publisher"].(string)),
 			Type:                     &extensionType,
 			TypeHandlerVersion:       pointer.To(extensionRaw["type_handler_version"].(string)),
-			AutoUpgradeMinorVersion:  pointer.To(extensionRaw["auto_upgrade_minor_version_enabled"].(bool)),
+			AutoUpgradeMinorVersion:  pointer.To(autoUpgradeMinorVersion),
 			ProvisionAfterExtensions: utils.ExpandStringSlice(extensionRaw["extensions_to_provision_after_vm_creation"].([]interface{})),
```

Comment:
`extensions_to_provision_after_vm_creation` also needs to be checked as it's not defined in `azurerm_linux_virtual_machine_scale_set ` either.

Reply:
Thanks for pointing this out @ms-zhenhua, we're going to merge this for now and follow up on this in a separate PR